_posts/en/newsletters/2024-05-15-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #302'
+permalink: /en/newsletters/2024/05/15/
+name: 2024-05-15-newsletter
+slug: 2024-05-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Release of utreexod beta:** Calvin Kim [posted][kim utreexo] to the
```

Comment:
I don't know if you'd like to link to this, but I learned about UtreeXO from this Bitcoin Explained (van Wirdum and Sjors) podcast episode (Oct 30, 2020), I thought it was very good: https://podcasts.apple.com/us/podcast/episode-15-utreexo/id1532957243?i=1000496616418

Reply:
I'd prefer not to link to podcasts.  If there's any good documentation about something, I think we should add links to it on our topic page.